src/main.ts

```diff
@@ -384,10 +384,12 @@ export class Loxone extends utils.Adapter {
             if (!stateChangeListener.opts?.selfAck) {
                 // Set ack timer before calling listener
                 stateChangeListener.ackTimer = this.setTimeout(
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
```

Comment:
i had to add a `@ts-ignore` here, because the type of `setTimeout` is not correct

Reply:
they defined the TimeoutCallback like so:
`export declare type TimeoutCallback = (args?: any[]) => void;`
but it should be like this:
`export declare type TimeoutCallback = (...args: any[]) => void;`

Reply:
If you think this is incorrect please raise it as an issue on that project.

Reply:
will do

Reply:
Link: https://github.com/ioBroker/ioBroker.js-controller/issues/2733